Casks/a/aigcpanel.rb

```diff
@@ -0,0 +1,24 @@
+cask "aigcpanel" do
+  arch arm: "arm64", intel: "x64"
+
+  version "0.12.0"
+  sha256 :no_check
```

Comment:
```suggestion
  sha256 arm:   "cf288a6f019d4eeba0110d7bbb3ad8c852c630eb818117ac8ba716d5aece0aa7",
         intel: "73324d009b8a79417d2146355b38c9e9ce17419c565a35924949169e98a3a453"
```

We can compute checksums since we are using a version URL.